angular/core/components/dashboard_page/dashboard_page.haml

```diff
@@ -1,33 +1,6 @@
 .lmo-one-column-layout
   %main.dashboard-page.lmo-row
     %section.dashboard-page__options{aria-label: "{{ 'dashboard_page.options.aria_label' | translate }}"}
```

Comment:
Think you can lose this section as well, and the translation that goes with it. :D